ming_sim/execution_pressure.py

```
@@ -356,16 +356,143 @@ def _province_open_counts(conn) -> Dict[str, int]:
+        "owners": list(owner_rows),
+        "arrival_rows": list(arrival_rows),
+    }
+
+
+def build_execution_two_axis_surface(
+    db,
+    turn: int = 0,
+    *,
+    transit_semantics: Sequence[object],
+) -> Dict[str, object]:
     """接口层纯函数：DB 状态 → 两轴清单（结构化 + TSV 文本）。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)